reference/OpenAPI.json

```diff
@@ -14341,40 +14365,30 @@
               "voucher"
             ]
           },
-          "applicable_to": {
-            "$ref": "#/components/schemas/6_res_applicable_to_object"
-          },
-          "inapplicable_to": {
-            "$ref": "#/components/schemas/6_res_inapplicable_to_object"
-          },
           "result": {
-            "description": "Specifies the redeemable's end effect on the order. This object is unique to each type of redeemable.",
-            "$ref": "#/components/schemas/19_obj_validations-true_result_gift_card"
-          },
-          "metadata": {
             "type": "object",
-            "description": "The metadata object stores all custom attributes in the form of key/value pairs assigned to the redeemable."
-          },
-          "categories": {
-            "$ref": "#/components/schemas/20_obj_category_object"
+            "properties": {
+              "error": {
+                "$ref": "#/components/schemas/e_error"
```

Comment:
To raczej outdated jest. W moim PR ja dodaje nowy obiekt `Error` możesz spróbować sobie to dociągnąć i podmienić może.